src/main/features/dashboard/helpers/claimStatusFlow.ts

```diff
@@ -0,0 +1,49 @@
+import { Claim } from 'claims/models/claim'
+import { Logger } from '@hmcts/nodejs-logging'
+
+const logger = Logger.getLogger('ClaimStatusFlow')
+
+export class ClaimStatusFlow {
+
+  static readonly flow: ClaimStatusNode = {
+    description: 'Claim Is Draft',
+    isValidFor: () => true,
+    next: [
+      {
+        description: 'Claim Issued',
+        isValidFor: (claim) => !claim.response,
+        dashboard: 'claim_issued',
+        next: []
+      }
+    ]
+  }
+  static decide (flow: ClaimStatusNode, claim: Claim): string {
+    if (flow.isValidFor(claim)) {
+      const nextPossibleConditions = (flow.next || []).filter(state => state.isValidFor(claim))
+      if (nextPossibleConditions.length > 1) throw new Error(`Two possible paths are valid for a claim, check the flow's logic`)
```

Comment:
@robertparkinson do we have a policy for TS as well as Java to have all code in `{` blocks `}`?

Reply:
Yeah we do keep everything inside { }

Reply:
should be fixed 👍